### DISCUSSION TITLE
ImportError: DLL load failed while importing _psycopg: The specified module could not be found (Postgres17 and create_engine)

### DISCUSSION THREAD

**Discussion by stephenjoly** [fastapi/sqlmodel#1206] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
$ c:/Users/myuser/Home/Development/venv2/Scripts/python.exe c:/Users/myuser/Home/Development/app/main.py
Traceback (most recent call last):
  File "c:\Users\myuser\Home\Development\app\main.py", line 30, in <module>
    engine=create_engine(DATABASE_URL)
  File "<string>", line 2, in create_engine
  File "C:\Users\myuser\Home\Development\venv2\Lib\site-packages\sqlalchemy\util\deprecations.py", line 281, in warned
    return fn(*args, **kwargs)  # type: ignore[no-any-return]
  File "C:\Users\myuser\Home\Development\venv2\Lib\site-packages\sqlalchemy\engine\create.py", line 599, in create_engine
    dbapi = dbapi_meth(**dbapi_args)
  File "C:\Users\myuser\Home\Development\venv2\Lib\site-packages\sqlalchemy\dialects\postgresql\psycopg2.py", line 690, in import_dbapi
    import psycopg2
  File "C:\Users\myuser\Home\Development\venv2\Lib\site-packages\psycopg2\__init__.py", line 51, in <module>
    from psycopg2._psycopg import (                     # noqa
    ...<10 lines>...
    )
ImportError: DLL load failed while importing _psycopg: The specified module could not be found.
(venv2)
```

### Description

Error when calling create_engine. Seems like it relys on psycopg2 which currently has an issue open for Python 13 support on Windows. I am having a lot of issues with this. Using Python 13 on windows 10. 

### Operating System

Windows

### Operating System Details

Windows 10

### SQLModel Version

Postgres 17

### Python Version

3.13.0

### Additional Context

I installed the C compiled from visual studio. Tried uninstalling and reinstalling countless times. Set up different instances of postgress. Psycopg2 seems to be the ultimate problem here and I am not sure what to do. Anyone know how to fix this?

**Comment by YuriiMotov** [comment, marked as answer]
`create_engine` loads different modules depending on what database URL you specify. 

Read more: https://docs.sqlalchemy.org/en/20/core/engines.html
